### PR TITLE
Fix Dart snippet references

### PR DESCRIPTION
**Pull request by ok300**
Fix a few typos and a copy-paste mistake in LNURL-withdraw.

### CODE REVIEW

File: snippets/dart_snippets/lib/lnurl_withdraw.dart [hunk 1]
```diff
@@ -3,21 +3,22 @@ import 'package:breez_sdk/bridge_generated.dart';
...
+      if (inputType is InputType_LnUrlWithdraw) {
+          int amountSats = inputType.data.minWithdrawable;
+          LnUrlCallbackStatus result = await lnurlWithdraw(
+              reqData: inputType.data,
+              amountSats: amountSats,
+              description: "comment",
+          );
+      }
+  } catch (error) {
+      // handle error
```
Comment (erdemyerebasmaz) [inline snippets/dart_snippets/lib/lnurl_withdraw.dart:21]: There's no need to update this latter part.

Comment (ok300) [reply snippets/dart_snippets/lib/lnurl_withdraw.dart:21]: True!

I was confused by the `lnurlPayUrl` in the first line, will revert.

File: snippets/dart_snippets/lib/lnurl_withdraw.dart [hunk 2]
```diff
@@ -3,21 +3,22 @@ import 'package:breez_sdk/bridge_generated.dart';
 
 Future<void> lnurlWithdraw() async {
   // ANCHOR: lnurl-withdraw
-  /// Endpoint can also be of the form:
-  /// lnurlp://domain.com/lnurl-pay?key=val
-  /// lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4excttsv9un7um9wdekjmmw84jxywf5x43rvv35xgmr2enrxanr2cfcvsmnwe3jxcukvde48qukgdec89snwde3vfjxvepjxpjnjvtpxd3kvdnxx5crxwpjvyunsephsz36jf
-  String lnurlPayUrl = "lightning@address.com";
+  // Endpoint can also be of the form:
+  // lnurlw://domain.com/lnurl-withdraw?key=val
```
Comment (erdemyerebasmaz) [inline snippets/dart_snippets/lib/lnurl_withdraw.dart:7]:
```suggestion
  /// Endpoint can also be of the form:
  /// lnurlw://domain.com/lnurl-withdraw?key=val
```

File: snippets/dart_snippets/lib/lnurl_withdraw.dart (hunk 1 above)
Comment (ok300) [reply snippets/dart_snippets/lib/lnurl_withdraw.dart]: Done.

File: snippets/dart_snippets/lib/lnurl_withdraw.dart (hunk 2 above)
Comment (ok300) [reply snippets/dart_snippets/lib/lnurl_withdraw.dart]: Not needed anymore after ae4bc95fb64d34677b60e9ebe8e6039153b5f554

**Review by erdemyerebasmaz — APPROVED** [review]

**Pull request merged**